cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -53,14 +53,21 @@ import org.ossreviewtoolkit.model.Package
 import org.ossreviewtoolkit.model.RemoteArtifact
 import org.ossreviewtoolkit.model.VcsInfo
 import org.ossreviewtoolkit.model.VcsType
+import org.ossreviewtoolkit.model.licenses.LicenseCategorization
```

Comment:
Commit message: I think it would be more telling if you explicitly state in the body

```
For a better overview, this change only implements the logic for the case
when one archive is created per project or package entity.
```